javascript/src/handleEnrichmentWebhook.js

```diff
@@ -26,28 +26,11 @@ module.exports = asyncHandler(async function handleEnrichmentWebhook(req, res) {
     `Received Enrichment webhook - https://app.transcend.io${req.body.extras.request.link}`,
   );
 
-  // Add new identifers
-  const signedRequestIdentifiers = {
-    email: [
-      createEnricherJwt({
-        value: 'test+enriched@transcend.io',
-      }),
-      createEnricherJwt({
-        value: 'test+access@transcend.io',
-      }),
-    ],
-    phone: [
-      createEnricherJwt({
-        countryCode: 'US',
-        number: '+18609066012',
-      }),
-    ],
-  };
-
   // Check if we should place a hold on this request
   const requestIdentifier = signedBody.value;
   const isFraudster = await checkIfFraudster(requestIdentifier);
   const hasLegalHold = await checkForLegalHold(requestIdentifier);
+  const nonce = req.headers['x-transcend-nonce'];
 
   // In this case, we are automatically cancelling requests from fraudsters.
   if (isFraudster) {
```

Comment:
What about this `res.json`?

Reply:
these are still allowed

Reply:
Can this happen async as well?

Reply:
not currently but probably could add cc @anotherminh

Reply:
Are we saying we want to support canceling the enrichment request async?

Reply:
Yeah

I think anyone having an async requirement for enrichment would have the same need when determining whether it ought to be canceled or on hold

Reply:
yeah good callout I'll add that as a requirement